src/components/TeamCard/TeamCard.scss

```diff
@@ -0,0 +1,70 @@
+@use "~stylesheets/typography";
+
+.Card {
+  animation: fadeIn 1s;
+  width: 292px;
```

Comment:
I would update this width to be 80-90%, and set a max-width to 292px which will allow this component to scale to the galaxy fold as well.